modules/OFConnectionManager2/module/src/cxn_instance.h

```diff
@@ -112,6 +112,8 @@ typedef struct controller_s {
                        * is disconnected? */
     uint32_t fail_count;  /* Increments each time a main cxn attempt fails;
                            * cleared when TCP connection is established */
+    uint32_t disconnected_count; /* Receive POLLHUP. See poll(3).
+                                  * Socket should be closed immediately. */
```

Comment:
suggest renaming this counter to "hup_count" or something else with a more descriptive name, because the controller will be disconnected due to any error